_pages/learn.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: splash
+permalink: /learn.html
+title: "Python Tutorials & Learning Resources for Scientists"
+classes: flowing
+header:
+  overlay_image: images/header.png
+  overlay_filter: 0.6
+scientists:
+  - image_path: images/pyopensci-people-at-computer.jpg
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "Our catalog of vetted open source tools makes it easier for scientists to find the trusted tools that they need to develop their open science workflows."
+    title: "Help scientists find the open science tools that they need"
+    url: https://www.pyopensci.org/python-packages.html
+    btn_label: View our growing list of accepted scientific Python packages
+    btn_class: btn--primary
+overview:
+  - excerpt: "pyOpenSci creates resources to help you navigate the Python packaging ecosystem with ease. All of our materials are community developed and go through extensive review."
+development:
+   - image_path: images/pyopensci-learn-community-created.png
+     alt: "alt here "
+     title: "Community Developed Python Tutorials for Scientists"
```

Comment:
We have inconsistent capitalization in our headers throughout the page, where some headers capitalize everything except for conjunctions, while others only capitalize the first word and proper nouns.

Reply:
i've gone back and forth on what is preferred. David preferred sentence case for headers. i kind of like capitals for main headers and sentence for sub headers (for skimmers)

what do you prefer? this is one of those things where i dont have a super strong opinion or preference.

Reply:
since you use AP style for your writing, I'd say let's use AP style for all headlines as well, just to keep things consistent, so: Capitalize only the first word of your headline and all proper nouns or abbreviations; all other words should be lowercase (e.g. “The people making North Dakota’s future bright”). (from https://coschedule.com/blog/headline-style-capitalization)

Reply:
ok! i wonder fi there is some sort of checker for this? i know i will mess it up in the future again. Even if it's in vscode it would help. do you know of any such tools?